src/ApiHandler.ts

```diff
@@ -701,8 +701,8 @@ export default class ApiHandler {
 				`Failed to get Metadata for block ${hash}, using latest.`
 			);
 			console.error(err);
-			this.specVersion = api.createType('u32', -1);
-			this.txVersion = api.createType('u32', -1);
+			this.specVersion = api.createType('u32', 0);
+			this.txVersion = api.createType('u32', 0);
```

Comment:
Same issue, these are valid spec and tx version numbers. If we're in an error condition we don't want to return results that might be wrong.